test/basic.test.js

```diff
@@ -598,5 +598,34 @@ test('basic prettifier tests', (t) => {
     t.is(arst, `[${epoch}] INFO  (on ${hostname}): hello world\n`)
   })
 
+  t.test('prettifies trace caller', (t) => {
+    t.plan(1)
+    const traceCaller = (instance) => {
+      const { symbols: { asJsonSym } } = pino
+      const get = (target, name) => name === asJsonSym ? asJson : target[name]
+
+      function asJson (...args) {
+        args[0] = args[0] || Object.create(null)
+        args[0].caller = '/tmp/script.js'
+        return instance[asJsonSym].apply(this, args)
+      }
+
+      return new Proxy(instance, { get })
+    }
+
+    const pretty = prettyFactory()
+    const log = traceCaller(pino({}, new Writable({
+      write (chunk, enc, cb) {
+        const formatted = pretty(chunk.toString())
+        t.is(
+          formatted,
+          `[${epoch}] INFO  (${pid} on ${hostname})</tmp/script.js>: foo\n`
```

Comment:
Why wouldn't there be a ` ` between `)<`?

Reply:
It's more compact but have no strong preference. Can add a space inbetween

Reply:
A prettifier isn't about compactness, it's about legibility. A space would be more appropriate.

Reply:
Will do 👍